plugins/catalog-react/src/components/EntityRefLink/EntityRefLink.tsx

```diff
@@ -80,7 +81,12 @@ export const EntityRefLink = forwardRef<any, EntityRefLinkProps>(
     const link = (
       <Link {...linkProps} ref={ref} to={entityRoute(routeParams)}>
         {children}
-        {!children && (title ?? formattedEntityRefTitle)}
+        {!children &&
+          (title ?? (
+            <Box maxWidth="200px">
```

Comment:
Hmm this `maxWidth` property is a little unfortunate here, as this size might change over time and in different places. Is there a way to say `maxWidth` is `1fr` or `100%` or something instead?

Reply:
I guess what I'm thinking is that the core component here just uses the available space, but the space limitation is set by the parent component that renders this in the header or what not.

Reply:
Just a ping on this @morales-martin